src/main/java/com/iheartradio/m3u8/Constants.java

```diff
@@ -83,6 +84,7 @@
     public static final String NO = "NO";
     private static final String INTEGER_REGEX = "\\d+";
     private static final String SIGNED_FLOAT_REGEX = "-?\\d*\\.?\\d*";
+    private static final String TIMESTAMP_REGEX = "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}Z";
```

Comment:
It looks like milliseconds and timezone are both optional. Can the regex be altered to allow for those? For reference:

https://tools.ietf.org/html/draft-pantos-http-live-streaming-17#section-4.3.2.6